test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -1115,6 +1115,49 @@ def test_qlinear_relu_int8_mixed_bf16(self):
         """
         self._qlinear_unary_cpu_test_helper(int8_mixed_bf16=True)
 
+    @skipIfNoDynamoSupport
+    @skipIfNoONEDNN
+    @skipIfRocm
+    def test_qlinear_gelu_cpu(self, int8_mixed_bf16=False):
```

Comment:
Can we extend and re-use `_qlinear_unary_cpu_test_helper`?

Reply:
Thanks for the comment and changed accordingly.